lib/ansible/plugins/inventory/openstack.py

```diff
@@ -172,8 +173,10 @@ def parse(self, inventory, loader, path, cache=True):
             else:
                 config_files = None
 
+            # Redict logging to stderr so it does not mix with output
+            # particular ansible-inventory JSON output
             # TODO(mordred) Integrate openstack's logging with ansible's logging
-            sdk.enable_logging()
+            sdk.enable_logging(stream=sys.stderr)
```

Comment:
LGTM. Should the inventory plugin be doing its own logging? @emonty Looks like logging was initially added in https://github.com/ansible/ansible/commit/d548c477c027441d12c10a83aed72424279da8c7

Reply:
I think it's fine if a plugin wants to log to a file, but the only supported data source the plugins have are the data from the inventory file, so it could add a `log_file` parameter and log there.

Integrating logging with Ansible logging in general would be more complex, and you can read more ranting about this in https://github.com/ansible/ansible/issues/51835